docs/blog/operator-migration-to-upstream-grafana.md

```diff
@@ -0,0 +1,82 @@
+But... the grafana-operator is a true side-gig for us, 3/4 of our current active maintainers don't work with Kubernetes
+day-to-day. We acknowledge that this is one of the major factors limiting development right now. Where some feature
+requests have been open for months at a time, with no one being able to implement them. With our limited time we tend to
+focus on fixing bugs and small quality-of-life improvements, however, we know that this is not enough and that the
+operator has much greater potential. After all, we only support a few core features, out of numerous possible ones in
+Grafana.
+
+So, the choice is pretty clear for us, considering the slow development speed, and our limited capacity but also wanting
+the operator to grow, in all aspects, community, featureset etc., we believe this to be a step in the right direction
+for everyone involved.
+
+## What does it mean for me(as a grafana-operator user)?
+
+Technically, not much will change, the grafana-operator will still be the grafana-operator you’ve gotten to know, love
+and have pulled millions of times (we see the stats 😁).
+
+We don’t plan on a new API version with this repo migration, at least not in the upcoming quarters, we’re really happy
+with how flexible our API is now, so we don’t really have a reason to change anything.
+
+We also don’t expect existing deployments to be affected in any capacity, and if for any reason that would happen, we'll
+be there to help out!
+
+## What about the maintainers?
+
+All current Grafana-Operator maintainers are planning on sticking around and continuing on with our involvement in the
+project, after all, this is a massive leap for the operator, and we’re all happy to be part of it!
+
+In all likelihood, new maintainers from the upstream Grafana community might join, so all-in-all, the maintainers might
+be more responsive to issues and pull requests!
+
+## Licensing?
+
+Speaking with the upstream Grafana folks that reached out to us, we've been assured that our current license will not
+pose an issue to anyone, and we've been promised that it will remain that way going forward as well.
+
+## What WILL change?
+
+Metadata mostly, we’ll likely switch a few key-values in our manifests to better reflect the actual state of the
+operator (i.e ownership, repository addresses, maintenance contacts etc)
+In general, house-keeping stuff that doesn't really affect users.
+
+The primary change will be the repo address on Github, we'll migrate
+the https://github.com/grafana-operator/grafana-operator to https://github.com/grafana.
+Github automatically redirects migrated addresses, so there's nothing to be concerned about on that end.
+
+We might end up moving a few other things around, like creating a new channel on the upstream Grafana slack, however,
+you'll definitely hear about any other changes before they happen! So, stay tuned!
+
```

Comment:
For people contributing to the grafana-operator the biggest change will be that you have to sign the grafana CLA https://grafana.com/docs/grafana/latest/developers/cla/

Reply:
When adding a section about this, I think it would be good to mention that the CLA is based on the Apache Software Foundation CLA which might put some developers (for whom CLAs are a sensitive topic) at ease.

Reply:
Added both! thanks!